.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: python
 
 python:
+  - 3.7
```

Comment:
travis doens't have 3.7 release version yet. You can try "3.7-dev"  which runs an old beta version, which may be good enough

Reply:
It finally has this version (need to use `dist: xenial` though). :blush: